style/index.css

```diff
@@ -2,4 +2,5 @@
 @import url('https://fonts.googleapis.com/css2?family=Libre+Franklin:wght@700&display=swap');
 @import url('base.css');
 @import url('layout.css');
-@import url('modules/sectionHeading.css');
\ No newline at end of file
+@import url('modules/sectionHeading.css');
+@import url('modules/scrollCarousel.css');
```

Comment:
here, and also at the end of `scrollCarousel.css`, we need newlines at the end of the file. otherwise every time somebody adds to a file we're going to have extra lines in the diff. I'll leave another comment with the best settings for VS code.